blocks/accordion/accordion.css

```diff
@@ -1,16 +1,12 @@
-.accordion details {
-    border: 1px solid #dadada;
-  }
-  
-  /* stylelint-disable-next-line no-descending-specificity */
-  .accordion details + details {
-    margin-top: 24px;
-  }
-  
-  .accordion details p {
-    margin-bottom: 0.8em;
-  }
-  
+.accordion-item-body p {
+  color: hsl(0deg 50% 25%);
```

Comment:
1. let's remove this color for the P element, we'll be adding it globally if at all . I'm going to see how the customer wants to proceed tomorrow. 
2. And in general, we shouldn't use HSL on the web. Results have unpredictable brightness, because the "L" (lightness) is not accurate with how our eyes work. Use the rgb or hex equiv.